package/boot/uboot-envtools/files/ath79

```diff
@@ -21,14 +29,42 @@ alfa-network,r36a|\
 allnet,all-wap02860ac|\
 arduino,yun|\
 buffalo,bhr-4grv2|\
+buffalo,wzr-bhr|\
```

Comment:
It's not an actual device. ```buffalo,bhr-4grv``` and ```buffalo,wzr-hp-g450h```

Reply:
I was modifying this late in night, and my string manipulation magic produced nonexistent compatible string. Fixed.